lightning/src/ln/channelmanager.rs

```diff
@@ -201,7 +201,9 @@ pub struct BlindedForward {
 	/// onion payload if we're the introduction node. Useful for calculating the next hop's
 	/// [`msgs::UpdateAddHTLC::blinding_point`].
 	pub inbound_blinding_point: PublicKey,
-	// Another field will be added here when we support forwarding as a non-intro node.
+	/// If needed, this determines how this HTLC should be failed backwards, based on whether we are
+	/// the introduction node.
+	pub failure: BlindedFailure,
```

Comment:
Hm, it should only be used in a failure context. Could you point me to where you're seeing that?

Reply:
By "used" I meant more that it is used in this struct even though described "if needed". I suppose the struct name is fine given how it is actually used, but the variable name `failure` makes it seem like something has already failed when in fact in this context it might not ever. I'm fine if there's not a better name.